package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "outerwhorld",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
```

Comment:
Some of these testing libraries likely won't be needed- do you guys want to write tests? Or should we remove these?